lib/pluginmanager/update.rb

```diff
@@ -34,14 +37,15 @@ def update_gems!
     # remove any version constrain from the Gemfile so the plugin(s) can be updated to latest version
```

Comment:
Are we actually locking plugins to versions throw the Gemfile? I don't think so, isn't? is this comment outdated? for now I understand we lock plugin version in the Gemfile.lock, isn't?